satpy/tests/reader_tests/test_olci_nc.py

```diff
@@ -274,3 +329,45 @@ def test_bitflags(self):
                              False, False,  True,  True, False, False, True,
                              False])
         assert all(mask == expected)
+
+
+class TestL1bBitFlags(unittest.TestCase):
```

Comment:
Same here. Remove the class and make the method a regular function.

Reply:
Sure, I will modify it.